e2e/cypress/integration/login/tutorial_navigation_and_links_spec.js

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+describe('Test Tutorial Navigation', () => {
+    before(() => {
+        cy.loginAsNewUserWithTutorial();
+    }); // before
+
+    it('MM-13989 / On13989: After login, the first page of the tutorial should display.', () => {
+        checkStepOne();
+    }); // should display the first step of the tutorial
+
+    it('A click of the middle circle should display the second page.', () => {
+        cy.get('#tutorialIntroCircle1').click();
+        checkStepTwo();
+    }); // should go to second step
+
+    it('A click of the third circle should display the third page.', () => {
+        cy.get('#tutorialIntroCircle2').click();
+        checkStepThree();
+    }); // should go to third step
+
+    it('A click of the first circle should display the first page again.', () => {
+        cy.get('#tutorialIntroCircle0').click();
+        checkStepOne();
+    }); // should go to first step
+
+    it('Clicking next on first step should show the second page', () => {
+        cy.get('#tutorialNextButton').click();
+        checkStepTwo();
+    }); // should go to second step
+
+    it('Clicking the icon should display the Mattermost download page.', () => {
+        // Source: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/testing-dom__tab-handling-links/cypress/integration/tab_handling_anchor_links_spec.js
+        //
+        // This fetches the contents of the appDownloadImage link and checks it without actually
+        // clicking it and opening it in a new tab as Cypress doesn't appear to handle this.
+
+        cy.get('#appDownloadImage').then(($a) => {
```

Comment:
No need to verify the download page this way. I think just verifying the href should be sufficient.

Reply:
Verified HREF but still testing for a 200 response when seeking the href as Saturnino Abril indicated on the QA channel: https://community-daily.mattermost.com/core/pl/cct5kpsc37g5udm8t9i679ae6a)